ngi_reports/stockholm/project_summary.py

```diff
@@ -114,22 +114,24 @@ def __init__(self, config, LOG, working_dir, **kwargs):
                 self.LOG.info('Sample {} is aborted, so skipping it'.format(sample_id))
                 self.project_info['aborted_samples'][sample_id] = {'user_id': sample.get('customer_name',''), 'status':'Aborted'}
                 continue
-            
+
             ## Basic fields from Project database
             self.samples_info[sample_id] = {'ngi_id': sample_id}
-            ## special characters should be removed 
-            self.samples_info[sample_id]['customer_name'] = self.to_ascii(sample.get('customer_name',''))
-            self.samples_info[sample_id]['total_reads'] = sample.get('details',{}).get('total_reads_(m)')
-            self.samples_info[sample_id]['reads_min'] = sample.get('details',{}).get('reads_min')
-            self.samples_info[sample_id]['preps'] = {}
-            
-            ## Check if a non-aborted sample is sequenced
-            if self.samples_info[sample_id]['total_reads'] == None:
+
+            ## get total reads if avialable or mark sample as not sequenced
+            try:
+                self.samples_info[sample_id]['total_reads'] = str(round(float(sample.get('details',{})['total_reads_(m)']), 2))
```

Comment:
Ah, I did wonder that - I was hoping that by specifying `f` in the float string, it would handle the `float()` conversion itself. Apparently not.